server.go

```diff
@@ -1553,6 +1553,13 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 		anchorPolicy := policy
 		anchorPolicy.BlobType |= blob.Type(blob.FlagAnchorChannel)
 
+		// Copy the policy for legacy channels and set the blob flag
```

Comment:
Not in this PR, but I don't think we should do policy initialization in `server.go`, it should be handled by wt so maybe moved into `NewManager`.